hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/upgrade/OMFinalizeUpgradeRequest.java

```diff
@@ -103,10 +110,13 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager, TermIn
           ozoneManager.getVersionManager().getMetadataLayoutVersion());
       LOG.trace("Returning response: {}", response);
     } catch (IOException e) {
+      exception = e;
       response = new OMFinalizeUpgradeResponse(
           createErrorOMResponse(responseBuilder, e), -1);
     }
 
+    auditLog(auditLogger, buildAuditMessage(OMAction.UPGRADE_FINALIZE,
+        new HashMap<>(), exception, userInfo));
```

Comment:
Can we add `termIndex` in auditMap as an additional info for audit message ?

Reply:
TermIndex will be handled as part of "[HDDS-10658](https://issues.apache.org/jira/browse/HDDS-10658). Add Object ID and Update ID to OM audit log messages" when this is merged.